fine_tuning/text_to_sql_inventory/prepare_dataset.py

```
@@ -0,0 +1,151 @@
+
+    idx = len(pairs)
+    while len(pairs) < 200:
+        cat = categories[idx % len(categories)]
+        wh = warehouses[idx % len(warehouses)]
+        status = statuses[idx % len(statuses)]
+        brand = brands[idx % len(brands)]
+
+        templates = [
+            (
+                f"How many {cat.lower()} products do we have?",
+                f"SELECT COUNT(*) AS count FROM products WHERE category = '{cat}';",
```

Ruff (0.15.20)

[error] Possible SQL injection vector through string-based query construction

(S608)